packages/beeq/src/components/progress/__tests__/bq-progress.e2e.ts

```diff
@@ -0,0 +1,58 @@
+import { newE2EPage } from '@stencil/core/testing';
+// import { computedStyle } from '../../../shared/test-utils';
+
+describe('bq-progress', () => {
+  it('should render', async () => {
+    const page = await newE2EPage({
+      html: '<bq-progress></bq-progress>',
+    });
+    const element = await page.find('bq-progress');
+
+    expect(element).toHaveClass('hydrated');
+  });
+
+  it('should have shadow root', async () => {
+    const page = await newE2EPage({
+      html: '<bq-progress></bq-progress>',
+    });
+    const element = await page.find('bq-progress');
+
+    expect(element.shadowRoot).not.toBeNull();
+  });
+
+  it('should render the progress bar with label', async () => {
+    const page = await newE2EPage({
+      html: '<bq-progress label></progress>',
+    });
```

Comment:
Perhaps we need to set the value so that we can test that the label has a text equal to `${value}%`, would that make sense?

Reply:
Yes, good idea.